### PR TITLE
Add missing fs-extra dependency

### PR DESCRIPTION
**Pull request by oschrenk**
Otherwise:

```
npm install -g git-http-server2
git-http-server2
module.js:472
    throw err;
    ^

Error: Cannot find module 'fs-extra'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/usr/local/lib/node_modules/git-http-server2/main.js:10:10)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
```

### CODE REVIEW

**Comment by Klortho** [comment]
Sorry! I have been seriously AWOL!

File: README.md [hunk 1]
```diff
@@ -43,21 +43,22 @@ For usage information:
...
-  -p, --port <port>   [env GIT_HTTP_PORT] port on which to listen
-  -r, --readonly      [env GIT_HTTP_READONLY] operate in read-only mode
-  -u, --updates       check for available updates and exit
-  -v, --version       print the version number and exit
+  -h, --help           print this message and exit
+  -i, --ip             [env GIT_HTTP_IP] IP address of the allowed client
+  -H, --host <host>    [env GIT_HTTP_HOST] host on which to listen
+  -p, --port <port>    [env GIT_HTTP_PORT] port on which to listen
+  -r, --readonly       [env GIT_HTTP_READONLY] operate in read-only mode
+  -a, --alllowcreation [env GIT_HTTP_ALLOWCREATION] allows pushing unknown repositories
```
Comment (Klortho) [inline README.md]: sp

Comment (oschrenk) [inline README.md]: Not sure what "sp" means. I fixed the typo in `--alllowcreation` if that was what you meant.

**Comment by oschrenk** [comment]
Didn't realize that I did all the changes on my master branch. If you need this to rebased or merged from a different branch let me know.

**Comment by Klortho** [comment]
Thanks!

**Pull request merged**